### ISSUE TITLE
Other language than English support

### ISSUE THREAD

**Issue by lens0021** [Underdoge/twitter-autoload-chrome#1]
First, thank you for the greatest extension!

https://github.com/Underdoge/x-autoload-chrome/blob/05674bdd1a62e9d8f6817407ef5b899c4097bca1/x_autoload.js#L27

The above line works for only English. Could it be extends, somehow?

**Comment by Underdoge** [comment]
Good idea, will see if we can make that more dynamic based on the local language, thank you
